packages/cli/src/commands/deploy.ts

```diff
@@ -16,6 +16,11 @@ export default class Deploy extends Command {
       description: 'Show state preview',
       default: false,
     }),
+    output: Flags.boolean({
```

Comment:
Do we have an issue or Slack discussion about `--output`? I'm just wondering how it should behave.

We should maybe add this to the README for `npx checkly deploy` as well https://github.com/checkly/checkly-cli/tree/b609488f438721e78f56a5a180d813e57a239268#npx-checkly-deploy

Reply:
@clample , no we don't. I found it useful because the `/next/project/deploy` returns all the project information and the `diff` applied. So, I found it useful to debug which resources were modified (created/updated/deleted).

Reply:
Thanks for clarifying. That makes sense.